public/samples/interop-samples/interop-examples.fs

```diff
@@ -0,0 +1,333 @@
+(*
+Interop Examples V 1.0 by Gary Bilkus March 2018
+
+This app is intended to demonstrate the various ways of interoperating with existing javascript.
+It goes progressively from a quick and dirty approach in which everything is untyped and you access the global namespace directly, through to use of modules and typed interfaces.
+
+The intention is to consolidate, in one reasonably small app, working examples of all the techniques available,from the perspective of someone who knows how to do things in javascript and wants to translate that into F#.
+The fable.io website does contain most of what's in here, but not necessarily in a way which is easy for a newbie to find.
+
+I am making no attempt to teach either javascript or F#.
+The index.html page which is the main page of this demo app contains a small script which defines various objects in javascript for us to use.
+
+http://fable.io/docs/compatibility.html
+and
+https://medium.com/@zaid.naom/f-interop-with-javascript-in-fable-the-complete-guide-ccc5b896a59f
+are (March 2018) the current main references, and worth reading.
+But there is some stuff in here which isn't in there, particularly the use of $ for function application
+and examples of Global, along with working examples of some things which are only briefly mentioned.
+*)
+namespace FableApp1
+(*
+Our first module demonstrates an approach to java interop in which we assume nothing at all about the underlying javascript objects, we just know how to use them from within existing javascript
+We can do the same with Fable - the syntax is a bit different, but the principles are the same.
+*)
+module Utils =
+ open Fable.Core
```

Comment:
Please use "4 spaces indentation or tab" here you are usinfg 1 space to indent the code and it's really hard to read